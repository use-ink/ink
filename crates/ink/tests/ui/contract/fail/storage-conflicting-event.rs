#![allow(unexpected_cfgs)]

#[ink::contract]
mod contract {
    #[ink(storage)]
    #[ink(event)]
    pub struct Contract {}

    impl Contract {
        #[ink(constructor)]
        pub fn constructor() -> Self {
            Self {}
        }

        #[ink(message)]
        pub fn message(&self) {}
    }
}

fn main() {}
