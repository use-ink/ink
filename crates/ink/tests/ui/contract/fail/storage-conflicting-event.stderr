error: encountered conflicting ink! attribute argument
 --> tests/ui/contract/fail/storage-conflicting-event.rs:6:11
  |
6 |     #[ink(event)]
  |           ^^^^^
